docs/introduction/renku.rst

```diff
@@ -1,46 +1,49 @@
 .. _renku:
 
-Renku
------
+Renku CLI
+---------
 
-The `renku` part of the Renku Project is a commandline interface (CLI) for annotating
-datasets and wrapping workflows with metadata for aiding iterative development and
-building a lineage of results. You can use this CLI from within an interactive environment
-launched from Renkulab or locally by installing the library onto your machine.
+The core of the Renku Project is the ``renku`` command-line interface (CLI), which offers
+tools for easily capturing your data science process as you work. With these tools, you
+can describe and annotate data and workflows, providing information that is used to
+build the lineage of your results, simplifying iterative development and making your
+work reproducible. The CLI can be used within Renkulab or locally, on your own machine.
 
-`renku` aims to be "git for data science" by applying version control to executed
-code tied together with the input files that were dependencies and the output files
-that result.
+The importance of version control for working with code is widely recognized.
+`renku` aims to be "git for research", by extending version control to encompass
+elements central to research: data and processes.
 
-Renku can be decomposed into to the following pieces that are exposed to the user.
+If that's too abstract, you can check out :ref:`firststeps` or see it in action in this video!
```

Comment:
Video? what video?? We should probably remove until further notice? :)

Reply:
oh. getting ahead of myself!